src/GraphQL/Types/Composite/AutoRegisteringInputObjectGraphType.cs

```diff
@@ -11,7 +11,9 @@ namespace GraphQL.Types
     /// Also it can get descriptions for fields from the XML comments.
     /// Note that now __InputValue has no isDeprecated and deprecationReason fields but in the future they may appear - https://github.com/graphql/graphql-spec/pull/525
     /// </summary>
-    public class AutoRegisteringInputObjectGraphType<TSourceType> : InputObjectGraphType<TSourceType>
+    public class AutoRegisteringInputObjectGraphType<
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties)]
+    TSourceType> : InputObjectGraphType<TSourceType>
```

Comment:
```suggestion
    public class AutoRegisteringInputObjectGraphType<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties)] TSourceType> : InputObjectGraphType<TSourceType>
```
I would not wrap the line here. For me single line is nmore readable than wrapped one.

Reply:
The same suggestion for all other usages.

Reply:
I'm talking about single line for generic arguments. I'm fine for such style:
![изображение](https://user-images.githubusercontent.com/21261007/212782874-ca4d8b76-766c-4134-9606-69855b6d9074.png)